### ISSUE TITLE
Drawer header label not announced

### ISSUE THREAD

**Issue by joe-replin** [adaptlearning/adapt-contrib-core#400]
### Subject of the issue/enhancement/features
The drawer heading span isn't being announced at anything at all in Firefox. The element receives focus, but nothing happens.

### Your environment
* FW v5.30.3
* Firefox v114.0.2
* JAWS 2023.2303.144

### Steps to reproduce
Tell us how to reproduce this issue.

### Expected behaviour
The drawer label should announce the global setting

### Actual behaviour
In this configuration of browser and screen reader, nothing is announced while the element receives focus.
